app/src/main/res/values/strings.xml

```diff
@@ -70,7 +70,7 @@
 
     <!-- TextsOnBlock -->
     <!-- Sequence -->
-    <string name="textOnBlocks.contentDefaultSec">0</string>
+    <string name="textOnBlocks.contentDefaultSec">0.0</string>
```

Comment:
Oh, I see... Thanks for pointing out.

Please let me leave a note for this. Suppose the following situation; 

1. The initial value on `TextView` (`NumberTextHolder`) is `0`
1. A user long touched (the trigger of `TextView`) the view
1. The user doesn't change the value and click `OK`
1. Then the value becomes `0.000` (yeah, I should have changed it to `0.000` not `0.0`:P) in the current implementation

I changed it because I thought this is a bit surprising for users but you told me that caused a bug. I think changing the initial value in `NumberTextHolder`s' constructors is neither a good idea... hmm...